generated/nixnet/nixnet.proto

```diff
@@ -0,0 +1,1239 @@
+  DB_PROPERTIES_ECU_FLEX_RAY_CONNECTED_CHS = 327700;
+  DB_PROPERTIES_ECU_FLEX_RAY_STARTUP_FRAME_REF = 84213777;
+  DB_PROPERTIES_ECU_FLEX_RAY_WAKEUP_CHS = 327699;
+  DB_PROPERTIES_ECU_FLEX_RAY_WAKEUP_PTRN = 327698;
+  DB_PROPERTIES_ECU_RX_FRM_REFS = 100990979;
+  DB_PROPERTIES_ECU_TX_FRM_REFS = 100990980;
+  DB_PROPERTIES_ECU_LIN_CONFIG_NAD = 327715;
+  DB_PROPERTIES_ECU_LIN_FUNCTION_ID = 327717;
+  DB_PROPERTIES_ECU_LIN_INITIAL_NAD = 327714;
+  DB_PROPERTIES_ECU_LIN_MASTER = 33882144;
+  DB_PROPERTIES_ECU_LINP2MIN = 17104934;
+  DB_PROPERTIES_ECU_LIN_PROTOCOL_VER = 327713;
+  DB_PROPERTIES_ECU_LINS_TMIN = 17104935;
+  DB_PROPERTIES_ECU_LIN_SUPPLIER_ID = 327716;
+  DB_PROPERTIES_ECU_NAME = 50659330;
+  DB_PROPERTIES_ECU_J1939_NODE_NAME = 151322665;
+  DB_PROPERTIES_ECU_J1939_PREFERRED_ADDRESS = 327720;
+  DB_PROPERTIES_LIN_SCHED_CLST_REF = 84279301;
+  DB_PROPERTIES_LIN_SCHED_COMMENT = 50724870;
+  DB_PROPERTIES_LIN_SCHED_CONFIG_STATUS = 393223;
+  DB_PROPERTIES_LIN_SCHED_ENTRIES = 101056513;
+  DB_PROPERTIES_LIN_SCHED_NAME = 50724866;
+  DB_PROPERTIES_LIN_SCHED_PRIORITY = 393219;
+  DB_PROPERTIES_LIN_SCHED_RUN_MODE = 393220;
+  DB_PROPERTIES_LIN_SCHED_ENTRY_COLLISION_RES_SCHED = 84344833;
+  DB_PROPERTIES_LIN_SCHED_ENTRY_DELAY = 17235970;
+  DB_PROPERTIES_LIN_SCHED_ENTRY_EVENT_ID = 458755;
+  DB_PROPERTIES_LIN_SCHED_ENTRY_FRAMES = 101122052;
+  DB_PROPERTIES_LIN_SCHED_ENTRY_NAME = 50790406;
+  DB_PROPERTIES_LIN_SCHED_ENTRY_NAME_UNIQUE_TO_CLUSTER = 50790408;
+  DB_PROPERTIES_LIN_SCHED_ENTRY_NC_FF_DATA_BYTES = 168230921;
+  DB_PROPERTIES_LIN_SCHED_ENTRY_SCHED = 84344839;
+  DB_PROPERTIES_LIN_SCHED_ENTRY_TYPE = 458757;
+  DB_PROPERTIES_PDU_CLUSTER_REF = 84410372;
+  DB_PROPERTIES_PDU_COMMENT = 50855938;
+  DB_PROPERTIES_PDU_CONFIG_STATUS = 524295;
+  DB_PROPERTIES_PDU_DEFAULT_PAYLOAD = 168296453;
+  DB_PROPERTIES_PDU_FRM_REFS = 101187590;
+  DB_PROPERTIES_PDU_MUX_DATA_MUX_SIG_REF = 84410377;
+  DB_PROPERTIES_PDU_MUX_IS_MUXED = 34078728;
+  DB_PROPERTIES_PDU_MUX_STATIC_SIG_REFS = 101187594;
+  DB_PROPERTIES_PDU_MUX_SUBFRAME_REFS = 101187595;
+  DB_PROPERTIES_PDU_NAME = 50855937;
+  DB_PROPERTIES_PDU_PAYLOAD_LEN = 524291;
+  DB_PROPERTIES_PDU_SIG_REFS = 101187589;
+}
+
+enum DBPropertyValue {
+  DB_PROPERTY_VALUE_UNSPECIFIED = 0;
+  DB_PROPERTY_VALUE_CA_NIO_MODE_CAN = 1;
+  DB_PROPERTY_VALUE_CA_NIO_MODE_CAN_FD = 2;
+  DB_PROPERTY_VALUE_CA_NIO_MODE_CAN_FD_BRS = 3;
+  DB_PROPERTY_VALUE_PROTOCOL_UNKNOWN = 4;
+  DB_PROPERTY_VALUE_PROTOCOL_CAN = 5;
+  DB_PROPERTY_VALUE_PROTOCOL_FLEX_RAY = 6;
+  DB_PROPERTY_VALUE_PROTOCOL_LIN = 7;
+  DB_PROPERTY_VALUE_PROTOCOL_ETHERNET = 8;
+  DB_PROPERTY_VALUE_APP_PROTOCOL_NONE = 9;
+  DB_PROPERTY_VALUE_APP_PROTOCOL_J1939 = 10;
+  DB_PROPERTY_VALUE_FRM_FLEX_RAY_CH_ASSIGN_A = 11;
+  DB_PROPERTY_VALUE_FRM_FLEX_RAY_CH_ASSIGN_B = 12;
+  DB_PROPERTY_VALUE_FRM_FLEX_RAY_CH_ASSIGN_AAND_B = 13;
+  DB_PROPERTY_VALUE_FRM_FLEX_RAY_CH_ASSIGN_NONE = 14;
+  DB_PROPERTY_VALUE_CLST_FLEX_RAY_SAMP_CLK_PER_P_0125US = 15;
```

Comment:
I'm not sure what this is supposed to be, but `P_0125US` looks a bit suspicious to me.